app/extension/bom/pom.xml

```diff
@@ -28,7 +28,7 @@
 
   <properties>
     <spring-boot.version>2.3.12.RELEASE</spring-boot.version>
-    <camel.version>2.23.2.fuse-790053</camel.version>
+    <camel.version>2.23.2.fuse-7_10_0-00016</camel.version>
```

Comment:
Let's do this in a separate PR, seems that we're hitting ENTESB-16583

Reply:
Unfortunately these two should be together, if syndesis uses 2.23.2.fuse-790053, then we should use `com.mongodb.MongoClient` in mongodb connector. Also, there are tests failures, because camel component wants to use the `com.mongodb.MongoClient`.

Reply:
Chicken or the egg, well, we can't do this upstream it seams. Not unless the PR for ENTESB-16583 gets merged and all dependencies get pushed to early access.